src/index.html

```diff
@@ -7,12 +7,57 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <title>Stars</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+
+    <div class="containerpai">
```

Comment:
Task says you should not add any other classes other than those defined in the **Checklist**